Mortgage-loan-calculator/src/app/admin-login/services/storage.service.ts

```diff
@@ -1,47 +0,0 @@
-import { Injectable } from '@angular/core';
-
-const TOKEN_KEY = 'auth-token';
-const USER_KEY = 'auth-user';
-
-@Injectable({
-  providedIn: 'root'
-})
-export class StorageService {
-  constructor() {}
-
-  clean(): void {
-    window.sessionStorage.clear();
-  }
-
-  public saveToken(token: string): void {
-    window.sessionStorage.removeItem(TOKEN_KEY);
-    window.sessionStorage.setItem(TOKEN_KEY, JSON.stringify(token));
-  }
-
-  public getToken(): string {
-    return JSON.parse(sessionStorage.getItem(TOKEN_KEY)|| '{}');
-  }
-
-  public saveUser(user: any): void {
-    window.sessionStorage.removeItem(USER_KEY);
-    window.sessionStorage.setItem(USER_KEY, JSON.stringify(user));
-  }
-
-  public getUser(): any {
-    const user = window.sessionStorage.getItem(USER_KEY);
-    if (user) {
-      return JSON.parse(user);
-    }
-
-    return {};
-  }
-
-  public isLoggedIn(): boolean {
```

Comment:
It should be the backend that responds if user still has active session. That is how it should be, but do not dwell on this too long right now.